app/services/idv/vendor_validated.rb

```diff
@@ -1,21 +0,0 @@
-module Idv
-  module VendorValidated
-    extend ActiveSupport::Concern
-
-    def vendor_validate
```

Comment:
They are not required to have the same API. The pattern we are using is that the controller calls a class that processes the form submission, and returns a `FormResponse` object that says whether or not the submission passed or failed, and what the errors were. How each class implements its `submit` method is up to the class. It just happens that these 3 IdV forms share similar code.

What I could do is move the calls to `vendor_validator.success?` that each class calls to a method in the parent Step class called `vendor_validation_passed?` or something like that.